packages/polimer/src/storeContext.ts

```diff
@@ -0,0 +1,81 @@
+import {PolimerEventHandler, PolimerStateFactory} from './polimerState';
+import {connect, sendUpdateToDevTools} from './tools/reduxDevToolsConnector';
+import { EventEnvelope, Router } from 'esp-js';
+
+type PolimerMapping = {
+    [eventName: string]: PolimerEventHandler<any>[];
+};
+
+type KeysOf<TStore, TValue> = {
+    [P in keyof TStore]: TValue
+};
+
+export type PolimerStore = {
+    [key: string]: any
+};
+
+export class PolimerStoreContext<T> {
+    private _polimerEventHandlers: PolimerMapping = {};
+
+    constructor(
+        private _store?: T,
+        private _autoCommit: boolean = true,
+        private readonly _storeName: string = 'ESP-JS'
+    ) {}
+
+    public initialize(router: Router, modelId: string) {
```

Comment:
Since now we have fat-pipe available, I would consider Polimer to start listening on all esp-js events, as opposed to manually deciding which ones youre interested about in the level above this.

At first glance this seems like a good place to do that logic.

Reply:
Sounds very sensible